_posts/2017/2017-10-06-epages-at-euruko.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "ePages at EuRuKo"
+date: "2017-09-27 09:34:02"
+image: blog-header/budapest-525857_1920.jpg
+image-position: top
+categories: events
+authors: ["Unai"]
+---
+
+Last week our team (Internal tools team) attended the European Ruby Conference in Budapest. Although we use more languages other than Ruby, we love it. It's the base of our team, so it felt natural be there.
+
+## An amazing location
+
+Budapest is awesome. It's a must see city, both during the day and at night. Gorgeous buildings, nice public transport and tasty food.
+
+## First day
+
+We arrived early to the registration, got a nice breakfast and the socialization started. One of the things that I love the most is how nice is this community. Everyone is friendly, everybody wants to share knowledge and listen to others, provide suggestions or just do some small talk.
+
+[Matz](https://github.com/matz) opened the conference, as it couldn't be other way. He gave us some hints about the future of Ruby and the team's focus and motivations.
+
+There was a really nice diversity of topics and also, just one track, so nobody missed anything. *Helping communities & products thrive by fostering empathy*, *How To Scale an Unscalable Service*, *Tensorflow Ruby API* or *Things I Learned the Hard Way Building a Search Engine* were some of the talk titles.
+
+Even thought Ruby is the central topic in most of them, it was nice to have other topics as empathy, system architecture or machine learning. For me that shows how mature and open minded is this community.
+
+## Lightning talks full of gems
+
+These talks were full of advices, from tips for juniors to system health monitoring. Some of the most interesting libraries for us were [tty](https://github.com/piotrmurach/tty) to ease the process of creating command line apps, [mini-check](https://github.com/workshare/mini-check) a small library to perform health checks or [reality](https://github.com/molybdenum-99/reality) an interesting experiment on ruby to process data and transform it to knowledge.
+
+## Euruko day two
+
+The second day started with a talk on [JRuby](http://jruby.org/) by [Charles Nutter](https://github.com/headius). It was awesome to see the issues, thought process and the numbers on other versions of Ruby and really intrigued by the performance and the good things that the JVM provides to Ruby, something that we are going to play with, for sure.
```

Comment:
It was awesome to get an insight into the issues, thought processes and the numbers of other Ruby versions. I was really intrigued by the performance and the advantages the JVM provides to Ruby. This is something that...